UDx-container/README.md

```diff
@@ -176,14 +178,49 @@ In addition to the tools such as `vsdk-make` and `vsdk-g++`, there is a `vsdk-ve
 
 The UDx container itself is not writable, so it creates and mounts a Docker volume called `verticasdk-data`. It also mounts the current working directory and your home directory in the container using the same names those directories have on the host machine. In addition, `vsdk-vertica` understands the `VSDK_MOUNT` and `VSDK_ENV` environment variables described in [environment variables](#environment-variables).
 
+
 ## Fetching the test Vertica server startup log
 
-`vsdk-vertica` launches a server that runs in the background in a container named `verticasdk`.  You can read the container log using the `docker logs` command:
+`vsdk-vertica` launches a server that runs in the background in a container named `verticasdk`.
+
+The command, when run, outputs the following message:
+
+```shell
+./vsdk-vertica
+Starting container...
+
+Run
+      docker logs verticasdk
+to view startup progress
+
+Don't stop container until above command prints 'Vertica is now running'
```

Comment:
I think it's silly to say not to stop vertica unless you also say why it's bad to stop Vertica.  I think in the case of docker containers, it's never bad to kill vertica at any time because the container can be re-built very easily.